### PR TITLE
Use system volume for spoken instructions

### PR DESCRIPTION
**Pull request by bsudekum**
This moves to use the system volume. This also allows the physical mute switch to be in the mute position and still get audio instructions. To mute instructions, the developer will have to set `volume?` to 0 or the user has to touch the mute button within the UI.

/cc @1ec5 @frederoni @ericrwolfe @JThramer

### CODE REVIEW

File: MapboxNavigation/RouteVoiceController.swift [hunk 1]
```diff
@@ -106,7 +96,7 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
             return
         }
         
-        rerouteSoundPlayer.volume = volume
+        rerouteSoundPlayer.volume = spokenInstructionVolume
```
Comment (frederoni) [inline MapboxNavigation/RouteVoiceController.swift]: AVAudioPlayer’s volume is relative to the hardware volume, using the same would mean that other audio sources would sound louder (or actually the opposite). For example, set hardware volume to 50%, and AVAudioPlayer.volume to AVAudioSession.outputVolume and AVAudioPlayer will use 25% of maximum volume. I'd suggest setting AVAudioPlayer.volume to 1 or even slightly higher.

Comment (frederoni) [reply MapboxNavigation/RouteVoiceController.swift]: Also, we have a conflicting PR here -> https://github.com/mapbox/mapbox-navigation-ios/pull/673 but I guess we can wait until this issue is ironed out.

Comment (bsudekum) [reply MapboxNavigation/RouteVoiceController.swift]: @frederoni if it's set to 1, while it always ignore the system volume?

Comment (frederoni) [reply MapboxNavigation/RouteVoiceController.swift]: @bsudekum no, it will strictly follow the system’s current volume.

**Pull request closed without merging**